.github/workflows/test.yml

```diff
@@ -1,9 +1,6 @@
 name: Test
```

Comment:
You forgot to change README.md tests/ -> app/
<img width="773" alt="image" src="https://user-images.githubusercontent.com/48014331/171046613-067d80ee-6c92-41d4-b662-b9043c84cbd4.png">